dictionaries/software-terms/dict/softwareTerms.txt

```diff
@@ -2419,6 +2419,7 @@ unindent
 unindented
 unindents
 uninstall
+uninstallation
```

Comment:
Updating just `src` would be [sufficient](https://github.com/streetsidesoftware/cspell-dicts/blob/main/dictionaries/software-terms/dict/README.md).